server/src/main/java/org/elasticsearch/index/mapper/IpFieldMapper.java

```diff
@@ -351,79 +346,10 @@ public static Query rangeQuery(
             return builder.apply(lower, upper);
         }
 
-        public static final class IpScriptDocValues extends ScriptDocValues<String> {
```

Comment:
Just for clarity, this all can be deleted because we use the other version now for `doc`?

Reply:
I moved it to [`IpDocValuesField.SortedSetIpSupplier`](https://github.com/elastic/elasticsearch/pull/81396/files#diff-5927e67432aff590f27d349fb19d63ea61f7aa16b45b847709a2095abce13ea1R142).  Seemed to make more sense there.